jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -64,10 +76,6 @@ def _on_viewer_data_changed(self, msg=None):
         self.dc_items = [lyr.label for lyr in self.app.data_collection
                          if layer_is_image_data(lyr)]
 
-        self.subset_items = [lyr.label for lyr in self.app.data_collection.subset_groups
-                             if lyr.label.startswith('Subset')]
-        self.bg_subset_items = ['Manual'] + self.subset_items

```

Comment:
So, you removed the ability for user to manually enter a background value?

Reply:
no, that is still all there!  Populating the options for the dropdown are just handled by the shared component, but if its set to 'Manual', the old behavior _should_ still work.